pkgs/development/python-modules/xsdata-pydantic/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  xsdata,
+  pydantic,
+}:
+
+buildPythonPackage rec {
+  pname = "xsdata-pydantic";
+  version = "24.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tefra";
+    repo = "xsdata-pydantic";
+    tag = "v${version}";
+    hash = "sha256-ExgAXQRNfGQRSZdMuWc8ldJPqz+3c4Imgu75KXLXHNk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    xsdata
+    pydantic
+  ];
+
```

Comment:
Can we execute the tests https://github.com/tefra/xsdata-pydantic/tree/main/tests ?

Reply:
As they also use pytest I guess we can also just add the `pytestCheckHook`?